.env.example

```diff
@@ -1,34 +1,75 @@
-# Docker settings
+## Docker settings
+
+# Traefik requires this file to be able to route the requests to the containers
 DOCKER_SOCKET=/var/run/docker.sock
 
-# Domain settings
+## Domain settings
+
+# Tag for images. It's stable by default
 DYO_VERSION=stable
+# Required for Traefik's certification resolution. It should be your
+# domain where dyrector.io will be available
 DOMAIN=example.com
+# Your server's timezone
 TIMEZONE=UTC
+# Required for Traefik's certification resolution
+# If there's an issue with the certificate, or when it expires,
+# letsencrypt will send a notificatiom to this e-mail address
 ACME_EMAIL=user@example.com
+# NodeJS services can run in two modes: production and development
+# These are the two values this key can have
 NODE_ENV=production
 
-# crux service settings
+## Crux service settings
+
+# You can specify how thorough logging will be
+# Options: verbose, debug, info, warning, error
+# The settings come in a hierarchic order,
+# meaning that in the order above they contain each other
+# Example: 'warning' contains 'error'
 LOG_LEVEL=debug
 
-# Database passwords
+## Database passwords
+
+# This value is the password to crux's database
 CRUX_POSTGRES_PASSWORD=Random_Generated_String
+# This value is the password to Kratos' database
 KRATOS_POSTGRES_PASSWORD=Random_Generated_String
 
-# External URL of the site https://example.com(:port if not 443)
+## External URL of the site https://example.com(:port if not 443)
+
+# This setting is to define where your
+# self-managed dyrector.io will be available
 EXTERNAL_URL=https://example.com
 
-# Cookie/JWT secrets
+## Cookie/JWT secrets
+
+# Secret to sign JWTs.
 CRUX_SECRET=Random_Generated_String
+# Secret to sign Kratos cookies
+# More details in Ory/Kratos documentation:
+# https://www.ory.sh/docs/kratos/reference/configuration
 KRATOS_SECRET=Random_Generated_String
 
-# Mailserver settings
+## Mailserver settings
+
+# The connection string for the mail server
+# The protocol can be SMTP or SMTPS
+# Example: protocol://smtp_user:smtp_password@mailserver_ip_or_domain:port
 SMTP_URI=smtps://username:password@mailserver.example.com:465
+# E-mail address for dyrector.io invitation links,
+# password resets and others
 FROM_EMAIL=from@example.com
+# E-mail sender name for dyrector.io invitation links,
+# password resets and others
 FROM_NAME=dyrector.io
 
-# Recaptcha secrets
-# In case you don't want to use ReCAPTCHA set DISABLE_RECAPTCHA to true
+## ReCAPTCHA secrets
+
+# In case you don't want to use ReCAPTCHA set DISABLE_RECAPTCHA to true.
+# Highly recommended to set DISABLE_RECAPTCHA to false
```

Comment:
Default value is not completely obvious, about the comments: Default: false => it is on.